### DISCUSSION TITLE
Teacher registration problems

### DISCUSSION THREAD

**Discussion by casacurta** [community/community#56282] [GitHub Education]
### Hi everyone,

Hello, I've tried several times to register as a teacher but I'm not having success.

I have already attached all possible documents but I still get the message that my profile has different information from the documents.

I've edited my profile several times trying to resolve this discrepancy, but I'm still unsuccessful.

If anyone can point me to what information I'm missing, I'd appreciate it.

I have been a teacher for 30 years and would love to be part of this community.

Thank you for your attention.

**Comment by samus-aran** [comment]
Hi @casacurta 👋 

Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying. 👈 I suspect this is required. 

Let us know how you get on 👍
